libs/cardano-ledger-pretty/src/Cardano/Ledger/Pretty/Conway.hs

```diff
@@ -311,7 +312,7 @@ instance PrettyA (Constitution era) where
       ]
 
 instance PrettyA (PParams era) => PrettyA (EnactState era) where
-  prettyA ens@(EnactState _ _ _ _ _ _ _) =
+  prettyA ens@(EnactState _ _ _ _ _ _ _ _) =
```

Comment:
I see this pattern often, whereas it could be replaced with: 
```
prettyA EnactState {..} = 
  ppRecord ...
```
But I wonder if it's kept like this on purpose so we get a warning when adding a new field?

Reply:
Nevermind, confirmed in the meeting